table.csv

```diff
@@ -72,6 +72,7 @@ aes-192,                        key,            0xa1,           draft,      192-
 aes-256,                        key,            0xa2,           draft,      256-bit AES symmetric key
 chacha-128,                     key,            0xa3,           draft,      128-bit ChaCha symmetric key
 chacha-256,                     key,            0xa4,           draft,      256-bit ChaCha symmetric key
+chacha20-poly1305,              multikey,       0xa5,           draft,      ChaCha20_Poly1305 encryption scheme
```

Comment:
One minor flag here - this is multi-byte so not a big deal, but adding this here doesn't leave any space for the plethora of authenticated encryption algorithms that it belongs with, I can easily imagine someone showing up and wanting to add a ton more but there being no space here. Maybe this belongs somewhere else where there's a bit more space to expand the AE list.

"multikey" seems a little limiting for this, but I don't have a better suggestion and I can't think of another way this might be consumed outside whatever you're doing.

Reply:
The multikey part is because Multikey objects can contain a chacha20-poly1305 symmetric key and tag. Multikeys can also contain an encrypted secret key that is encrypted using chacha20-poly1305.

I'm happy to move this to a 2-byte value.

Reply:
It already encodes to a 2-byte varint so it's not a major blocker, it just doesn't allow room for additional AE schemes in this block, the only way it really fits is because it's next to chacha-* but it's actually quite different than plain chacha. Not a big deal, just trying to think ahead.

Reply:
Want me to move it? I'm happy to do that.

Reply:
hey, how about we just bump it to `0xa00`, there's a huge block up there if we end up needing it, and 0xa00 encodes to the same number of varint bytes (2) as `0xa5`.

Reply:
ok, let me do that